File: Output/c/easy/FirstUniqueCharacterInAString.c
/**https://leetcode.com/problems/first-unique-character-in-a-string/ */
//Given a string s, find the first non-repeating character in it and return its index. If it does not exist, return -1.


int firstUniqChar(char * s){
    int count[26] = {0};
    int i = 0;
    while(s[i] != '\0'){
        count[s[i] - 'a']++;
        i++;
    }
    i = 0;
    while(s[i] != '\0'){
        if(count[s[i] - 'a'] == 1){
            return i;
        }
        i++;
    }
    return -1;

}

    

File: Tool/PreUtil/src/output/python/easy/FirstUniqueCharacterInAString.py
#https://leetcode.com/problems/first-unique-character-in-a-string/
#Given a string s, find the first non-repeating character in it and return its index. If it does not exist, return -1.


class Solution(object):
    def firstUniqChar(self, s):
        """
        :type s: str
        :rtype: int
        """ 
        if len(s)==0:
            return -1
        if len(s)==1:
            return 0
        for i in range(0,len(s)):
            if s.count(s[i])==1:
                return i
        return -1

    